src/main/resources/config.properties

```
@@ -13,3 +13,10 @@ pom.version=${project.version}
 # The language of the educational content
 # See https://github.com/elimu-ai/model/blob/master/src/main/java/ai/elimu/model/enums/Language.java
 content.language=ENG
+
+# Create free API key using https://www.covalenthq.com/platform/#/auth/register/
+# It will be used to get ETH transaction data from https://www.covalenthq.com/docs/api/#get-/v1/{chain_id}/address/{address}/transactions_v2/
+covalentApiKey=USE_YOUR_API_KEY
+
+# Used to fetch latest transactions
+donationAddress=0x883753Beab357A2c29f3766C6ad158e72A78ce51
```

Add newline character:

```suggestion
donationAddress=0x883753Beab357A2c29f3766C6ad158e72A78ce51

```